src/bootstrap/config.rs

```diff
@@ -765,13 +765,14 @@ impl Config {
 
         // set by build.rs
         config.build = TargetSelection::from_user(&env!("BUILD_TRIPLE"));
-        let manifest_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        let base_path =
+            output(Command::new("git").arg("rev-parse").arg("--show-toplevel")).trim().to_owned();
```

Comment:
I think this will break vendoring. We should fallback to `CARGO_MANIFEST_DIR` if git exits with an error.

Reply:
you marked these comments as resolved, but I don't see any code changes?